Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -254,23 +254,27 @@ public func onNativeTouch(
     else { return }
 
     Task { @MainActor in
+        guard let screenScale = UIScreen.main?.scale else {
+            return
+        }
```

Comment:
This seems much more likely to be the cause of the crash. I don't know when it'd be possible to receive touch events when `UIScreen` is `nil`, but it seems like a no brainer not to allow that specifically.